components/Form.js

```diff
@@ -22,39 +27,145 @@ export default function Form({ handleSubmit }) {
         required
       />
       <CharacterCounter maxLength={30} counter={count} />
-      <label htmlFor="description">Description:</label>
-      <textarea
+      <Label htmlFor="description">Description:</Label>
+      <DescriptionTextarea
         id="description"
         name="description"
         maxLength="100"
         onChange={(event) => setCountDescription(event.target.value.length)}
       />
       <CharacterCounter maxLength={100} counter={countDescription} />
-      <label htmlFor="imageFile">Add your sketch:</label>
-      <input
+      {router.pathname === "/create-new-project" && <SelectCategories />}
+      <Label htmlFor="imageFile">Add your sketch:</Label>
+      <Input
         type="file"
         name="imageFile"
         id="imageFile"
         size={10000}
         required
       />
-      <Button type="submit" disabled={statusUpload}>
-        Save
-      </Button>
-      <p>{statusUpload}</p>
+      <ButtonWrapper>
+        <Button type="button" onClick={() => router.back()}>
+          <StyledCancelIcon /> Cancel
+        </Button>
+        <Button type="submit" disabled={statusUpload}>
+          <StyledSaveIcon /> Save
+        </Button>
+      </ButtonWrapper>
+      <LoadingStatus>
+        {statusUpload && <Spinner />}
+        {statusUpload}
+      </LoadingStatus>
     </StyledForm>
   );
 }
 
+const StyledForm = styled.form`
+  display: grid;
+  padding: 0 1em;
+`;
+
+const Label = styled.label`
+  font-weight: 600;
+  font-size: 1.2em;
+  padding: 0.5em 0;
+`;
+
+const styledInputs = css`
+  background-color: var(--color-list-items-white);
+  border: none;
+  border-radius: 2em;
+  padding: 0.7em;
+  &:focus {
+    border-color: var(--color-buttons-yellow);
+  }
+`;
+
+const Input = styled.input`
+  ${styledInputs}
+  min-height: 5vh;
+  &:last-of-type {
+    border: none;
+    background: none;
+    font-size: 0.9em;
+  }
+`;
+
+const DescriptionTextarea = styled.textarea`
+  ${styledInputs}
+  min-height: 10vh;
+  resize: none;
+`;
+
+const StyledSvg = css`
+  width: 2em;
+  height: 2em;
+`;
+
+const StyledSaveIcon = styled(SaveIcon)`
+  ${StyledSvg}
+`;
+
+const StyledCancelIcon = styled(CancelIcon)`
+  ${StyledSvg}
+`;
+
+const ButtonWrapper = styled.div`
+  display: flex;
+  gap: 1em;
+  justify-content: space-evenly;
+  position: relative;
+  top: 3em;
+`;
+
 const Button = styled.button`
-  width: 150px;
-  height: 40px;
-  justify-self: flex-end;
-  margin: 10px 0;
+  display: flex;
+  align-items: center;
+  gap: 0.5em;
+  margin: 0;
+  border: none;
+  border-radius: 2em;
+  cursor: pointer;
+  &:active {
+    position: relative;
+    top: 1px;
+  }
+  &:last-of-type {
+    background-color: var(--color-buttons-yellow);
+    padding: 0.2em 3.5em;
+  }
+  &:first-of-type {
+    background-color: var(--color-list-items-white);
+    color: var(--color-project-inactive);
+    fill: var(--color-project-inactive);
+    padding: 0.2em 3em;
+  }
 `;
```

Comment:
I think you may have buttons with almost the same styling. 🤔 You could think about creating a styled button component with different variants to reuse some of your buttons. See our tourio app to get inspiration: https://github.com/mbosselmann/tourio/blob/main/components/StyledButton.js